pkg/dss/models/geo.go

```diff
@@ -2,17 +2,27 @@ package models
 
 import (
 	"errors"
+	"fmt"
 	"time"
 
 	"github.com/golang/geo/s2"
+	"github.com/golang/protobuf/ptypes"
 	"github.com/interuss/dss/pkg/dss/geo"
+	"google.golang.org/protobuf/proto"
+
+	// TODO: all of the uses of ridpb should use protos that can be used
```

Comment:
Let's remove this TODO as this project must accept whatever the standard APIs are -- we don't have the ability (as a code TODO) to change the RID and SCD object structures.  We can advocate for compatible object structures the the standard development, but the standards are consensus-based documents not controlled by this project.

Reply:
Note that we could still use common protos while sticking to the standard's spec